packages/fuels/tests/contracts.rs

```diff
@@ -1775,3 +1775,42 @@ async fn contract_custom_call_build_without_signatures() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+#[should_panic(
+    expected = "method not found (this should never happen): InvalidType(\"Token limit (1) reached while Encoding. Try increasing it.\")"
```

Comment:
This is weird. Why are we getting the `method not found` message when reaching the token limit?